src/WebdriverClassicDriver.php

```diff
@@ -1,21 +1,1345 @@
 <?php
 
-/*
+/**
  * This file is part of the Behat\Mink.
  * (c) Konstantin Kudryashov <ever.zet@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
+ *
+ * @noinspection PhpLanguageLevelInspection
```

Comment:
why this ? Let your phpstorm configure its language level based on the composer.json and it should be enough.